src/FederatedPeg.sln

```diff
@@ -25,7 +25,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Sidechain Generation", "Sid
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.FederatedPeg.IntegrationTests", "Stratis.FederatedPeg.IntegrationTests\Stratis.FederatedPeg.IntegrationTests.csproj", "{A8CA48FA-A725-412D-9EC4-1BA7BA7177B2}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Stratis.FederatedSidechains.AdminDashboard", "Stratis.FederatedSidechains.AdminDashboard\Stratis.FederatedSidechains.AdminDashboard.csproj", "{2D99612E-84FE-4D46-AB3C-B6A4C08828E9}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.FederatedSidechains.AdminDashboard", "Stratis.FederatedSidechains.AdminDashboard\Stratis.FederatedSidechains.AdminDashboard.csproj", "{2D99612E-84FE-4D46-AB3C-B6A4C08828E9}"

```

Comment:
Is this related to this PR?

Reply:
as I commented on sidechain channel

@here
adding the dns project to the solution VS changed AdminDashboard GUID for its project type
from
`Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Stratis.FederatedSidechains.AdminDashboard", "Stratis.FederatedSidechains.AdminDashboard\Stratis.FederatedSidechains.AdminDashboard.csproj", "{2D99612E-84FE-4D46-AB3C-B6A4C08828E9}"`
to
`Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.FederatedSidechains.AdminDashboard", "Stratis.FederatedSidechains.AdminDashboard\Stratis.FederatedSidechains.AdminDashboard.csproj", "{2D99612E-84FE-4D46-AB3C-B6A4C08828E9}"`

is it fine to apply this change or should i manually change it back?
basically the project type of admindashboard has been migrated from
FAE04EC0-301F-11D3-BF4B-00C04F79EFBC
to
9A19103F-16F7-4668-BE54-9A1E7A4F7556
here someone reported this "issue" and it seems its correct to change that
https://github.com/dotnet/project-system/issues/3079